index.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv').config()
+const express = require('express');
+const app = express();
+const port = 3000;
```

Comment:
Think about when you will use react in the same time

Reply:
done